menhir-recovery-generator/main.ml

```diff
@@ -63,11 +63,11 @@ let () =
           ) (Lr1.transitions st);
         fprintf ppf "Reductions:\n";
         List.iter (fun (t,ps) ->
-            let p : production = List.hd ps in
+            let p : production = List.hd [ps] in (* FIXME *)
```

Comment:
Did the API change? Why are we using `List.hd`, `List.mem`, or `List.map` on single-element lists?